app/ui-utils/client/lib/MessageAction.js

```diff
@@ -355,4 +356,26 @@ Meteor.startup(async function() {
 		order: 17,
 		group: 'menu',
 	});
+
+	MessageAction.addButton({
+		id: 'reaction-list',
+		icon: 'emoji',
+		label: 'Reaction_list',
+		context: ['message', 'message-mobile', 'threads'],
+		action() {
+			const { msg: { reactions } } = messageArgs(this);
+			console.log(reactions);
+
+			modal.open({
```

Comment:
Ideally the `ReactionList` component can be a modal itself. Ask @ggazzo for help about triggering it from a Blaze component, I think he did something like that for the user cards.